catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/query/geofeature/GazetteerFeatureServiceTest.java

```diff
@@ -0,0 +1,110 @@
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.catalog.ui.query.geofeature;
+
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.codice.ddf.spatial.geocoder.GeoResult;
+import org.codice.ddf.spatial.geocoding.GeoEntry;
+import org.codice.ddf.spatial.geocoding.GeoEntryQueryException;
+import org.codice.ddf.spatial.geocoding.GeoEntryQueryable;
+import org.geotools.geometry.jts.spatialschema.geometry.DirectPositionImpl;
+import org.geotools.geometry.jts.spatialschema.geometry.primitive.PointImpl;
+import org.junit.Before;
+import org.junit.Test;
+import org.opengis.geometry.DirectPosition;
+
+public class GazetteerFeatureServiceTest {
+    private static final GeoEntry GEO_ENTRY_1 = new GeoEntry.Builder().name("Philadelphia")
+            .latitude(40)
+            .longitude(-71)
+            .featureCode("PPL")
+            .population(1000000)
+            .alternateNames("")
+            .build();
+
+    private static final GeoEntry GEO_ENTRY_2 = new GeoEntry.Builder().name("Camden")
+            .latitude(40)
+            .longitude(-70)
+            .featureCode("PPLC")
+            .population(10000000)
+            .alternateNames("")
+            .build();
+
+    private static final List<GeoEntry> QUERYABLE_RESULTS = Arrays.asList(GEO_ENTRY_1, GEO_ENTRY_2);
+
+    private static final String TEST_QUERY = "example";
+
+    private GazetteerFeatureService gazetteerFeatureService;
+
+    private GeoEntryQueryable geoEntryQueryable;
+
+    @Before
+    public void setUp() {
+        geoEntryQueryable = mock(GeoEntryQueryable.class);
+        gazetteerFeatureService = new GazetteerFeatureService();
+        gazetteerFeatureService.setGeoEntryQueryable(geoEntryQueryable);
+    }
+
+    @Test
+    public void testGetSuggestedFeatureNames() throws GeoEntryQueryException {
+        final int maxResults = 2;
+        doReturn(QUERYABLE_RESULTS).when(geoEntryQueryable)
+                .query(TEST_QUERY, maxResults);
+
+        List<String> results = gazetteerFeatureService.getSuggestedFeatureNames(TEST_QUERY,
+                maxResults);
+        assertThat(results.get(0), is(equalTo(GEO_ENTRY_1.getName())));
```

Comment:
✏️  You can use the [`contains()`](http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/Matchers.html#contains(E...)) matcher for this: `assertThat(results, contains(GEO_ENTRY_1.getName(), GEO_ENTRY_2.getName()))`